x-pack/plugins/security_solution_serverless/server/endpoint/services/set_package_policy_flag.ts

```diff
@@ -87,11 +89,17 @@ async function processBatch(
                 meta: {
                   ...meta,
                   serverless: true,
+                  billable: false,
```

Comment:
why do you set `billable` to `false` here and update it below? could this be set with `isBillablePolicy()` here right away

Reply:
I'm using `false` here as the default value. We can't use `isBillablePolicy` here yet since it requires the entire policy to be passed in including the new `serverless` value.